.travis.yml

```diff
@@ -6,11 +6,15 @@ env:
     - TYPE=test
     - TYPE=check
 
+cache:
+  directories:
+    - $HOME/.m2
+
 before_install:
   - cd org.aposin.licensescout.core
 install:
   - mvn -DskipTests=true -Dmaven.javadoc.skip=true install

```

Comment:
@pfistererm - could we change the mvn install command to mvn package to use the cache functionality of Travis? If it is not required to run the full installation but just check that it compiles/package here (before running the test/check matrix) then we should close this PR.

Reply:
@pfistererm  i assigned you as the reviewer, please check this. thx

Reply:
I think for checking if compiling is OK and doing checks we do not need the install target of Maven. 'package' should be sufficient. I think we can try it. Alternatively (if there are problems with using 'package') we could try 'verify'.